src/services/user.service.ts

```diff
@@ -0,0 +1,65 @@
+import {Users} from '../loaders/database';
+import {Op} from 'sequelize';
+import bcrypt from 'bcrypt';
+import {v4} from 'uuid';
+
+const SALT_ROUNDS = 10;
+const RETURN_ATTRIBUTES = ['id', 'login', 'age'];
+
+export default class UserService {
+  static async getSuggestedUsers(
+    loginSubstring: string,
+    limit: number | undefined
+  ) {
+    return await Users.findAll({
```

Comment:
as I mentioned in the controller file - endpoint returns passwors, deletedAt

Reply:
fixed